app/supabase-provider.tsx

```diff
@@ -17,14 +17,16 @@ export default function SupabaseProvider({
 }: {
   children: React.ReactNode;
 }) {
-  const [supabase] = useState(() => createPagesBrowserClient());
+  const [supabase] = useState(() => createClient());
```

Comment:
```suggestion
  const supabase = createClient();
```

You no longer need to wrap this in `useState` as the `@supabase/ssr` package implements its own singleton pattern 👍

Reply:
Thanks I was going to ask about this!